contents/blog/feature-flags-vs-configuration.md

```diff
@@ -0,0 +1,135 @@
+
+Developers assume configurations will be permanent. Values might change, but they plan for that key and its associated value to always be important. This means versioning is critical for some configurations. Versioning helps debug potential issues related to code controlled by configs.
+
+> **Note:** Configurations can be environment variables, but they aren’t exclusively. They can be key-value pairs stored anywhere in your stack.
+
+### What is remote configuration?
+
+A remote configuration, or remote config, is a configuration that can be set and changed remotely. This means a developer can change the value for users without needing to change any code. [Firebase](https://firebase.google.com/docs/remote-config) popularized this, but it also exists for [Unity](https://unity.com/products/remote-config) and many other services.
+
+Remote config is especially relied on for mobile apps. This is because mobile apps must go through an especially long review process, which can take 24-48 hours. If an issue pops up, this review process delays shipping a fix. If an app uses remote configs, it can mitigate (turn off) the problem feature immediately.
+
+## What is a feature flag?
+
+[Feature flags](/docs/feature-flags) function nearly the same as configurations. They are a key that returns a value when called with a service, but they often include detail on the user as well.
+
+> **Note:** For simplicity, we assume you are using an external feature flag service. Of course, you could (and many do) build your own feature flag tool. This might include some of the functionality below.
+
+The big difference compared to configurations is feature flags’ expanded functionality:
+
+- Contains more data, like multiple variants and JSON.
+- Connects to usage and user data.
+- Provides simplified creation UI and permissioning.
+
+To access these, feature flag services often connect to external services like CDPs, product analytics, and data warehouses. They get data from these sources to customize which users get specific values when calling the flag.
+
+Feature flags are almost always remotely configurable, and because they use an external service, developers can modify them at run time. No need to reset or recompile, just call the external service again. These services also provide a simplified UI, creating a centralized location for modifying feature flags and enabling quick rollouts and rollbacks.
+
+An example use case of a feature flag is rolling a new feature into beta. A conditional statement wraps the beta feature to check the value of the flag. The feature flag service evaluates if the user is part of the beta, and either grants or denies the user access.
+
+![Feature flag video](../images/blog/feature-flags-vs-configuration/feature-flag.mp4)
+
+As showcased by this use case, feature flags are often meant to be temporary. Once you test and fully release a feature, you can remove the flag from the code. The failure to remove it creates technical debt.
```

Comment:
```suggestion
As showcased by this use case, feature flags are often meant to be temporary. Once you test and fully release a feature, you can remove the flag from the code. The failure to remove it can create technical debt.
```